dbt_project.yml

```diff
@@ -10,8 +10,16 @@ models:
     intermediate:
       +materialized: ephemeral
     staging:
-      +materialized: view
+      +materialized: ephemeral
```

Comment:
This is technically a breaking change as the old views are now going to be stale. I would call this out in the CHANGELOG as a breaking change and inform customers that the previous staging views will no longer be used.

Reply:
Also, don't forget to add the link to the PR in the CHANGELOG for all these changes.